src/Microsoft.DotNet.Wpf/src/Shared/MS/Internal/SecurityHelper.cs

```diff
@@ -143,23 +136,10 @@ internal static CodeAccessPermission CreateUserInitiatedRoutedEventPermission()
         /// Safe     - The method denies the caller access to the full exception object.
         /// </SecurityNote>
         [SecuritySafeCritical]
-        internal static bool CallerHasUserInitiatedRoutedEventPermission()
-        {
-            try
-            {
-                CreateUserInitiatedRoutedEventPermission().Demand();
-            }
-            catch (SecurityException)
-            {
-                return false;
-            }
-            return true;
-        }
-
-        static UserInitiatedRoutedEventPermission _userInitiatedRoutedEventPermission = null;
+        internal static bool CallerHasUserInitiatedRoutedEventPermission() { return true; }
```

Comment:
Only one site of use, which can be replaced with `return true` directly (thus enabling deletion of this method)